presentations/2023-09-07_coffee_and_coding_functions/index.qmd

```diff
@@ -0,0 +1,391 @@
+---
+title: "Repeating Yourself with Functions"
+subtitle: "Coffee and Coding"
+author: "[Sally Thompson](mailto:sally.thompson37@nhs.net)"
+date: 2023-09-07
+date-format: "DD MMMM YYYY"
+
+format:
+  revealjs:
+    theme: [default, su_presentation.scss]
+    self-contained: true
+    transition: none
+    chalkboard: false
+    preview-links: auto
+    slide-number: false
+    auto-animate: true
+    width: 1920
+    height: 1080
+    footer: |
+      This is my first attempt at a Quarto presentation!
+      
+execute:
+  echo: true
+---
+
+```{r setup}
+#| echo: false
+
+library(here)
+library(purrr)
+library(lubridate)
+library(zoo)
+library(dplyr)
+library(janitor)
+library(ggpubr)
+library(StrategyUnitTheme)
+
+# drive loc
+zdrive_loc <- "Z:/Strategic Analytics/Projects 2022/1056 WMAP Cardiac/SD Modelling/"

```

Comment:
Aha, I don't have access to this folder so I can't rebuild the slides. Would it be possible to create a small dataframe within this code chunk, or perhaps read in a small demo csv that you add to the `presentations/2023-09-07_coffee_and_coding_functions/` folder?

Reply:
csv file added and the code altered